docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -471,6 +471,31 @@
           "default": false
         }
       }
+    },
+    "extensions": {
+      "type": "object",
+      "description": "Extends various parts of the Grafana UI with commands or links.",
+      "additionalProperties": false,
+      "properties": {
+        "links": {
+          "type": "array",
+          "description": "Expose a page link that can be used by Grafana core or other plugins to navigate users to the plugin",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "description": {
+                "type": "string"
+              },
+              "path": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
```

Comment:
i didn't realize changes here were still being made by hand 😢 

Needs to be in the `plugindef.cue` as well, and we gotta start generating this json schema file out of the cue. Multiple sources of truth :(

Reply:
Once we have a single source of truth (CUE) - do we still need to generate this (by then) "build output" and keep it in our version control? I guess in that case it will just create confusion.

Reply:
Then it would be auto-generated and possibly named plugin.schema_gen.json or similar to denote it's generated

Reply:
I have updated it but I agree with @marefr and @leventebalogh